website/content/docs/getting-started.md

```diff
@@ -71,8 +71,20 @@ $ heron submit local \
   ~/.heron/examples/heron-examples.jar \ # The path of the topology's jar file
   com.twitter.heron.examples.ExclamationTopology \ # The topology's Java class
   ExclamationTopology # The name of the topology
+
+  INFO: Launching topology 'ExclamationTopology'
+
+  # etc
+  [2016-06-07 16:44:07 -0700] com.twitter.heron.scheduler.local.LocalLauncher INFO: \
+  For checking the status and logs of the topology, use the working directory \
+  /Users/{username}/.herondata/topologies/local/{role}/ExclamationTopology # working directory
+
+  INFO: Topology 'ExclamationTopology' launched successfully
+  INFO: Elapsed time: 3.409s.
 ```
 
+Note it may be helpful to know your working directory, as outputted above.
```

Comment:
Instead of saying that it would be helpful to know it, let's note that the working dir is logged which can be useful for troubleshooting.